azure/spoke/variables.tf

```diff
@@ -96,21 +102,12 @@ variable "subnets" {
 
 variable "hub_peering" {
   description = "Config for peering to the hub network."
-  type = object({
+  type = map(object({
```

Comment:
A map? Are we peering the spoke to multiple hubs? What's the rationale here?

Reply:
The rational is that the network module takes a map as an input so it simplifies things